src/Microsoft.Extensions.Logging.EventSource/EventSourceLoggerFactoryExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging.EventSourceLogger;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <summary>
+    /// Extension methods for the <see cref="ILoggerFactory"/> class.
+    /// </summary>
+    public static class EventSourceLoggerFactoryExtensions
+    {
+        /// <summary>
+        /// Adds an event logger that is enabled for <see cref="LogLevel"/>.Information or higher.
+        /// </summary>
+        /// <param name="factory">The extension method argument.</param>
+        public static ILoggerFactory AddEventSourceLogger(this ILoggerFactory factory)
```

Comment:
I assume you are talking about "filter spec" that governs what data goes through the bridge from ILogger to EventSource. 

I am not sure about this suggestion. I think it would be confusing for the API user to tell the difference between the spec passed through EventSource commands vs. the spec passed here. Which one wins if there is a conflict? Having just one place to specify the spec is beneficial. Event more importantly, the spec is given by the consumer of the data. The AddEventSourceLogger() API is for the producer of the data. The producer should not care what spec is used.